lib/ansible/modules/monitoring/logstash_plugin.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+"""
+Ansible module to manage elasticsearch plugins
+(c) 2017, Loic Blot <loic.blot@unix-experience.fr>
+
+This file is part of Ansible
+
+Ansible is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Ansible is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: logstash_plugin
+short_description: Manage Logstash plugins
+description:
+    - Manages Logstash plugins.
+version_added: "2.3"
+author: Loic Blot (@nerzhul)
+options:
+    name:
+        description:
+            - Name of the plugin to install.
```

Comment:
All `description:` (apart from `short_description:`) must be full sentences (capital letters & fullstops)